e2e-tests/cypress/e2e/SendCoins.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Send coins
+  As a user
+  I want to send and receive GDD
+  I want to see transaction details on overview and transactions pages
+
+  # Background:
+  #   Given the following "users" are in the database:
+  #     | email                  | password | name               |
+  #     | bob@baumeister.de      | Aa12345_ | Bob Baumeister     |
+  #     | raeuber@hotzenplotz.de | Aa12345_ | Räuber Hotzenplotz |
+  
+  Scenario: Send GDD to other user
+    Given the user is logged in as "bob@baumeister.de" "Aa12345_"
+    And the user navigates to page "/send"
+    When the user fills the send form with "raeuber@hotzenplotz.de" "<amount>" "Some memo text"
```

Comment:
I am bit perplex cause I don't understand the Examples block at the end of the Scenario.
Would it not be nicer to have:
```suggestion
    When the user fills the send form with "<receiverName>" "<amount>" "<description>"
```